api/lua/README.md

```diff
@@ -0,0 +1,23 @@
+# Lua API for PlayDate
+
+High-level lua API built on-top of [playdate-sys][].
+
+
+## Usage
+
+```rust
+use playdate_lua::Lua;
+
+let lua = Lua::new();
+
+// TODO
```

Comment:
Could be great to fill this with some minimal example like "how to register my c-lua-function" if it's not complex. If not - okay for initial crate state and first version 🤷🏻‍♂️.